<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_installator</class>
 <widget class="QDialog" name="create_installator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>809</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Путь</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_copy">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Путь</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_delete"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Путь</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_install">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Путь</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_icon">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pb_fileDialogCopy">
           <property name="text">
            <string>проводник</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_fileDialogInstall">
           <property name="text">
            <string>проводник</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_filedialog_install">
           <property name="text">
            <string>проводник</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_filedialog_icon">
           <property name="text">
            <string>проводник</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pb_copy">
           <property name="text">
            <string>Скопировать файлы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_delete">
           <property name="text">
            <string>Удалить файлы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_install">
           <property name="text">
            <string>Создать инсталлятор</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_icon">
           <property name="text">
            <string>Выбрать картинку</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layValid">
       <item>
        <widget class="QPushButton" name="pbValidator">
         <property name="text">
          <string>проверка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblValid">
         <property name="text">
          <string>----------------------------------------</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lbl_icon">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Место для значка приложения</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Название приложения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_ApplicationName"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Выберите файлы для создания ярлыков..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listLinks">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="movement">
        <enum>QListView::Snap</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Выберите файлы для загона в автозагрузку..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listAutoLoading">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="movement">
        <enum>QListView::Snap</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
